packages/slate-editor/src/modules/nodes-hierarchy/tests/Tables.test.tsx

```diff
@@ -303,4 +303,245 @@ describe('nodes-hierarchy / Tables', () => {
 
         expect(editor.children).toEqual(expected.children);
     });
+
+    it('should remove empty row', () => {
```

Comment:
> Tables.test.tsx

Ideally, this test should live inside `slate-tables` package directory. As it's only testing the code inside `slate-tables` package.

Reply:
There is a set-up for jsx and types in this package, i would leave this tests here for now and will think how to share elements bindings among pacakges

Reply:
Alright. Let's address this later.

> how to share elements bindings among packages

For the record, I'd better not overcomplicate the setup with shared bindings. A local partially duplicated setup inside `slate-tables` can be simpler if nothing is shared.